### PR TITLE
Fix eslint errors

### PR DESCRIPTION
**Pull request by eduarmreyes**
This PR fixes all the ESLint errors.

### CODE REVIEW

File: src/components/global/H1.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import React from "react"
```
Comment (aalejandromr) [inline src/components/global/H1.js:1]: So when not using functional components, we don't need to  import React even though we are basically setting a component? did you test this?

File: src/components/iOSDownloadButton.js [hunk 2]
```diff
@@ -2,8 +2,13 @@ import React from "react"
 import iOSDownloadImage from "src/images/iOsBadgeBanner.svg"
 
 const IOSDownloadButton = props => (
-  <a href="https://example.com" target="_blank" {...props}>
-    <img src={iOSDownloadImage}></img>
+  <a
```
Comment (aalejandromr) [inline src/components/iOSDownloadButton.js:5]: Can you get me this Eslint settings?

File: src/components/global/H1.js (hunk 1 above)
Comment (eduarmreyes) [reply src/components/global/H1.js:1]: yes, you do not need React, I tested these changes

File: src/components/iOSDownloadButton.js (hunk 2 above)
Comment (eduarmreyes) [reply src/components/iOSDownloadButton.js:5]: yes

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

**Review by aalejandromr — APPROVED** [review]

**Pull request merged**